doc/man1/timew-tags.1.adoc

```diff
@@ -10,3 +10,9 @@ timew-tags - display a list of tags
 == DESCRIPTION
 Displays all the tags that have been used by default.
 When a filter is specified, shows only the tags that were used during that time.
+
+== CONFIGURATION
+
+**tags.**__<tag>__**.color**::
+Assigns a specific foreground and background color to a tag, instead of the default color palette determined by your current theme. +
```

Comment:
I would use `+` only in rare circumstances. Start a new paragraph instead.

Reply:
Alright. I opted to remove the linebreak instead to keep it consistent with the other configuration options, since the line lengths seem similar enough.